apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/GrunnlagRoutes.kt

```diff
@@ -13,20 +13,17 @@ import no.nav.etterlatte.libs.common.SAKID_CALL_PARAMETER
 import no.nav.etterlatte.libs.common.grunnlag.NyeSaksopplysninger
 import no.nav.etterlatte.libs.common.grunnlag.Opplysningsbehov
 import no.nav.etterlatte.libs.common.grunnlag.opplysningstyper.Opplysningstype
-import no.nav.etterlatte.libs.common.hentNavidentFraToken
 import no.nav.etterlatte.libs.common.kunSystembruker
 import no.nav.etterlatte.libs.common.person.Folkeregisteridentifikator
-import no.nav.etterlatte.libs.common.person.InvalidFoedselsnummerException
 import no.nav.etterlatte.libs.common.sakId
-import no.nav.etterlatte.libs.common.withFoedselsnummer
 import no.nav.etterlatte.libs.common.withSakId
 
 fun Route.grunnlagRoute(
     grunnlagService: GrunnlagService,
     behandlingKlient: BehandlingKlient,
 ) {
-    route("grunnlag") {
-        get("{$SAKID_CALL_PARAMETER}") {
+    route("sak/{$SAKID_CALL_PARAMETER}") {
```

Comment:
Skulle det vært grunnlag/sak/$...?

Reply:
Flyttet til `Application.kt` for å wrappe både person- og saksgrunnlag.